minishift_sanity.py

```diff
@@ -92,90 +92,164 @@ def test_ms_start(self):
         else:
             self.fail("Minishift start failed")
             provision_path = None
+    """
     
-    """        
+    def test_ms_setup_cdk(self):
```

Comment:
Tests new downstream command which installs RHEL iso, ocp binary and config minishift to use them natively. Will rewrite this to be able to run also in upstream.